service/shorty.go

```diff
@@ -25,6 +25,7 @@ const (
 	StatusRedirected      = "redirected"
 	StatusBlocked         = "blocked"
 	StatusExpired         = "expired"
+	StatusLimitReached    = "limit reached"
```

Comment:
this status should be machine usable, could you replace de ` ` with `_` 
```suggestion
	StatusLimitReached    = "limit_reached"
```